src/scraping/start_for_life/scrape_sitemap.py

```diff
@@ -0,0 +1,89 @@
+"""
+Get all URLs from the NHS https://www.nhs.uk/start-for-life/site-map/
+
+Usage:
+
+First navigate to the scraping folder
+$ cd src/scraping/start_for_life
+
+Then run the script
+$ poetry run python scrape_sitemap.py
+
+The final output file is stored in data/sitemap.csv
+with the following columns:
+    - Title: Title of the page
+    - URL: URL of the page
+
+"""
+import csv
+import logging
+
+import requests
+
+from bs4 import BeautifulSoup
+
+
+# Headers to be used when scraping
+HEADERS = {
+    "User-Agent": "Data collection for the purpose of research. For questions, reach out to karlis.kanders@nesta.org.uk"
+}
+
+# URL with the website you want to scrape
+BASE_URL = "https://www.nhs.uk/start-for-life/site-map/"
+# Path to your output CSV file with titles and URLs
+OUTPUT_PATH = "data/sitemap.csv"
+
+
+def scrape_urls(base_url: str, csv_filename: str, timeout: float = 10) -> None:
```

Comment:
`scrape_urls` would benefit from [separating](https://en.wikipedia.org/wiki/Separation_of_concerns) it to multiple functions. For example, we could have three functions:

- `get_url()` calls the `base_url` and checks for status like `response.raise_for_status()` (ideally in a try/except block)
- `parse_sitemap()`: The `beautifulsoup` stuff
- `write_to_file()`: Write to a CSV

Another benefit is that `get_url` would also be used in `scrape_startforlife.py`

Reply:
minor: Should `timeout` be a float or int?